native/core/src/execution/jni_api.rs

```diff
@@ -249,7 +254,14 @@ fn prepare_datafusion_session_context(
 
     let runtime = RuntimeEnv::new(rt_config).unwrap();
 
-    let mut session_ctx = SessionContext::new_with_config_rt(session_config, Arc::new(runtime));
+    let state = SessionStateBuilder::new()
+        .with_config(session_config)
+        .with_runtime_env(Arc::new(runtime))
+        .with_default_features()
```

Comment:
default features needed to use the planner?

Reply:
Let me see if I can remove that

Reply:
Removing this causes test failures. We were previously calling the following method to create the context and this also enabled default features.

```rust
    pub fn new_with_config_rt(config: SessionConfig, runtime: Arc<RuntimeEnv>) -> Self {
        let state = SessionStateBuilder::new()
            .with_config(config)
            .with_runtime_env(runtime)
            .with_default_features()
            .build();
        Self::new_with_state(state)
    }
```

Reply:
since we using a planner only perhaps this should be enough

```
.with_query_planner(SessionStateDefaults::default_expr_planners())
```
?